src/ucp/api/ucp.h

```diff
@@ -660,16 +741,27 @@ typedef struct ucp_ep_params {
     uint64_t                field_mask;
 
     /**
-     * Destination address; the address must be obtained using
-     * @ref ucp_worker_get_address.
-     * This field is mandatory for filling (along with its corresponding bit
-     * in the field_mask - UCP_EP_PARAM_FIELD_REMOTE_ADDRESS).
-     * The ucp_ep_create routine will return with an error if the address isn't
+     * Destination address; if a client-server connection establishment
```

Comment:
if the UCP_EP_PARAM_FIELD_REMOTE_ADDRESS flag is set but this address isn't specified and flags is 0, we shouldn't return an error? (ucp_address_unpack will fails in this case)